changelogs/CHANGELOG-v0.3.0.md

```diff
@@ -0,0 +1,38 @@
+The v0.3.0 release is our second official minor release of the the bpfd project :tada
```

Comment:
:tada is showing up as :tada in the generated markdown.

Reply:
I'll add it manually after 👍